swarm/index.md

```diff
@@ -4,6 +4,13 @@ keywords: docker, swarm,  clustering
 title: Docker Swarm
 ---
 
+This section refers to standalone Docker Swarm. If you use Docker 1.12 or higher,
+[Swarm mode](../engine/swarm/index.md) is integrated with Docker Engine, and you
+should use Swarm mode instead.
```

Comment:
If there are things you can't do with the builtin version we should probably enumerate them. I'm open to suggestions.